services/LocationService.ts

```diff
@@ -1,6 +1,13 @@
 import Location from '../classes/location';
 
-function locationToGooglePlaceString(location: Location): string {
+export function locationToGooglePlaceString(location: Location): string {
   return `${String(location.getLatitude())}, ${String(location.getLongitude())}`;
 }
+export function parseGoogleMapLocation(location: any) {
```

Comment:
this function should be in the googleMapsAdapter.ts class. The purpose of this class is to parse googlemaps data into our data that fits our application. In reality googleMapsAdapter should have been a service, but its what we have.
Adding it to the location service does make sense, but because we use the adapter design pattern, we need to be consistent with our logic

Reply:
resolved